rad/src/operators/array.rs

```diff
@@ -58,6 +62,44 @@ pub fn map(input: &RadonArray, args: &[Value]) -> Result<RadonTypes, RadError> {
     Ok(RadonArray::from(result).into())
 }
 
+pub fn sort(input: &RadonArray) -> Result<RadonArray, RadError> {
+    // Sort not applicable if not homogeneous
+    if !input.is_homogeneous() {
+        Err(RadError::UnsupportedOpNonHomogeneous)?;
+    }
+    // Get discriminant and values
+    let disc = input.inner_type();
+    let value = input.value();
+
+    let mut result = Vec::new();
+    // Distinguish each discriminant case
+    if disc == discriminant(&RadonTypes::from(RadonString::default())) {
+        result = value
+            .iter()
+            .cloned()
+            .sorted_by(|a, b| match (a, b) {
```

Comment:
There is a `sorted_by_key` function which is more clear in this case, you just need to return the inner string.

```suggestion
            .sorted_by_key(|v| match v {
```

Reply:
It turns out this is cannot be done because of lifetimes...